api/package.json

```
@@ -134,7 +134,7 @@
     "@types/ms": "0.7.31",
     "@types/multer": "^1.4.5",
     "@types/multistream": "2.1.1",
-    "@types/node": "^14.17.27",
+    "@types/node": "^20.1.2",
```

as discussed, we should use a `^18.14.whatever` version here instead that the latest, to keep this in sync (ish) with the pinned version of the node engine, e.g.

```suggestion
    "@types/node": "^18.14.2",
```